Patches/Rim-Sheks!/Patch_PawnKinds_Sheks.xml

```diff
@@ -0,0 +1,193 @@
+			<li>Rim-Shek</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/PawnKindDef[defName="Shek_Knight"]/combatPower</xpath>
+					<value>
+						<combatPower>60</combatPower>
+					</value>
+				</li>
+
+				<!-- SHEK OUTLANDERS???Why do they override the vanilla defs?-->
+
+<!--  			 <li Class="PatchOperationAddModExtension">
+  				<xpath>Defs/PawnKindDef[defName="Villager" or defName="Town_Guard"]</xpath>
+  				<value>
+      					<li Class="CombatExtended.LoadoutPropertiesExtension">
+        					<primaryMagazineCount>
+          					<min>1</min>
+          					<max>3</max>
+        					</primaryMagazineCount>
+        					<sidearms>
+          					<li>
+            					<generateChance>0.5</generateChance>
+            					<sidearmMoney>
+              					<min>20</min>
+              					<max>120</max>
+            					</sidearmMoney>
+            					<weaponTags>
+              					<li>CE_Sidearm_Melee</li>
+            					</weaponTags>
+          					</li>
+        					</sidearms>
+     					 </li>
+
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Pirate"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>2</min>
+								<max>3</max>
+							</primaryMagazineCount>
+							<sidearms>
+								<li>
+									<sidearmMoney>
+										<min>40</min>
+										<max>160</max>
+									</sidearmMoney>
+									<weaponTags>
+										<li>CE_Sidearm_Tribal</li>
+										<li>CE_Sidearm_Melee</li>
+									</weaponTags>
+								</li>
+							</sidearms>
+						</li>
+					</value>
+				</li>
+  </li> -->
+
+				<!-- SHEK Pirates-->
+
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Scavenger_Shek"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>2</min>
+								<max>3</max>
+							</primaryMagazineCount>
+							<sidearms>
+								<li>
+									<sidearmMoney>
+										<min>40</min>
+										<max>160</max>
+									</sidearmMoney>
+									<weaponTags>
+										<li>CE_Sidearm_Tribal</li>
+										<li>CE_Sidearm_Melee</li>
+									</weaponTags>
+								</li>
+							</sidearms>
+						</li>
+					</value>
+				</li>
+
+				<!-- SHEK WARRIOR -->
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/PawnKindDef[defName="Shek_Warrior" or defName="Shek_Merchant" or defName="Shek_Vassal"]</xpath>
+					<value>
+						<li Class="CombatExtended.LoadoutPropertiesExtension">
+							<primaryMagazineCount>
+								<min>2</min>
+								<max>3</max>
+							</primaryMagazineCount>
+							<shieldMoney>
+								<min>100</min>
+								<max>150</max>
+							</shieldMoney>
+							<shieldTags>
+								<li>TribalShield</li>
+							</shieldTags>
+							<shieldChance>0.5</shieldChance>
```

Comment:
I don't think the merchants or vassals need shields.

Reply:
Vassals seem to feature prominently in some raids.